### DISCUSSION TITLE
What are the hard parts of Playabl?

### DISCUSSION THREAD

**Discussion by jongrim** [Playabl-io/playabl#197] [General]
Hello everyone! Now that Playabl has been in use for a bit, I can see people developing strong opinions on some of the designs and experiences. I recently saw one person mentioning that just finding a game had too high a mental cost to use the site often. That's obviously not in line with the goals, so I'd like to ask for help in identifying what the mentally tough parts are when using the site. What are the things that just feel tougher than they should be? And what do you need in those moments to make it easier?

Feel free to add comments with feedback and thoughts. I do kindly ask to try and avoid duplicates and use reactions to plus 1 or thumbsup what you agree with.

Thanks everyone!

**Comment by dhmontgomery** [comment]
It's very minor, but my primary point of annoyance is that when you sign up for a multi-session series, you have to manually download an ICS file for each one to add it to your calendar.

**Comment by ChaoticBlade5** [comment]
Creating an account is difficult if you try any way besides using your Google Account. 

The "home page" when logged in isn't different from where you are logged out, it makes people think they're doing something wrong and log back out.

There is no FAQ or tutorial about how to use the site. Documentation may have the information, but the word "Documentation" makes it seem it's for experts on Playabl.

When you click on the categories tab, it creates a drop down menu instead of taking you to the categories page. There should be direct links on the "home page" to the categories. As it reduce the number of clicks a user needs to make in order to join a game.

**Comment by machinic** [comment]
Event Creation (for first time event creators especially)

When creating an event for the very first time it is not clear that the Save button will make the event public. This could be mitigated either with some help text and/or changing the button label from "Save" to "Publish" which conveys a stronger sense of what comes next. There are other changes that could be made that would provide even sturdier guardrails, such as having the user confirm the publish action either by modal or by making save draft and publish two explicit actions, but most of those seem to be a greater change than what's necessary to gently guide our understanding whereas a button label change or some help text seems more appropriate to the issue.

The next time around, most creators will have learned their lesson if they were caught off guard by saving their event to work on later and seeing it go live. But it is also a mostly avoidable thing with some additional support for the user.

**Comment by AnnieCM** [comment]
(For context - I’m pretty new to the site and community; I’m usually on mobile, browsing in between doing other things. I’m usually following a direct link to a game someone’s mentioned elsewhere, so I at least have some initial information.)

Landing on the game page - if I’m not logged in, it tells me I don’t have access and must be a member of the community. So I have to know/remember that that’s due to not being logged in. Then, I don’t see an obvious login button. Expanding the menu and selecting Profile will let me log in, but that took some exploration to find out. 

Once I’m logged in, I’m on the Profile page. So now I’ve lost the game I was interested in. I can use the Back button, but then I’m back to the screen saying I don’t have access. Refreshing the page appears to fix that, or I can navigate around until I find the game again.

The next things I’m interested in are game/tone/description - to confirm if this is something I’m interested in - and dates/times, to see if I can make the scheduling work. On mobile, the title and informational icons take up almost a full screen, and the entry for each session takes up the full screen. So I have to scroll a good way to get to the description and the full schedule - especially for something with a lot of sessions. 

Joining the sessions and adding to calendar worked pretty well for me. (Granted I haven’t done something with many sessions yet!) It would be nice to have a “what to expect” message or email. It would also be helpful to have a clear way to contact the GM in case of questions. 

If I want to look for games without a direct link - Games/Browse with filters works pretty well. There are a few speed bumps here:

There’s no easy way to identify new entries. So, even with filters, I have to scroll past all the games I’ve either decided against or already signed up for to see if there’s anything new. (I guess another approach here would be a way to mark/hide games I’ve decided against.)

I’m not sure if this Browse page includes series that have already started; if it does, I don’t see a way to tell if that’s the case. (I did eventually find that the Community Calendar page has this info, but that page takes more digging to find.)

If I click into a specific game page and then back, I’m back at the top of the list, and I’ve lost my filters. So I have to reset the filters and remember/scroll back to where I was, if I didn’t remember to open in a different tab.

The other place I ran into trouble was the Shared Hearth Event page. I got a bit lost between the overview/calendar/more information pages. On the Overview page, the ribbon with upcoming games seems like the obvious place to look for what I might be interested in, but it doesn’t have filters, doesn’t show many specifics about each game, and has the same issue where clicking into a page and then back goes back to the beginning of the ribbon. The event calendar is easier to navigate, once I figured out the difference, but that took some trial and error. The calendar also doesn’t have all the filters the Browse page does (specifically, the time filter would be nice).

**Comment by machinic** [comment]
Event Participants "reserved for...":

Managing "one seat being held for foo" type of situations is difficult when you can't connect that foo (who also has an account in playabl) to the event. As an event creator, I'm willing to do a little bit of work to add that person to the game officially so that if they need to open door / cancel prior to the game the wait list can work for me on that seat as well. If the person for whom the seat is being reserved has an account / is a member of the community in playabl, and I'm reserving a seat for them, I presumably have some details at hand that will let me connect their account to the event without exposing the membership data to a creator wholesale. Being able to officially add folks for whom a seat is reserved will also help the event communicate an accurate count of players to people browsing the event (who may not dive all the way into the description to see what a creator has written about it in a text field).
